protostar.py

```diff
@@ -79,6 +79,12 @@ def cairo_file(file_path: str) -> Path:
 
 
 cmd_init_parser = root_subparsers.add_parser("init")
+cmd_init_parser.add_argument(
+    "--existing",
+    default=False,
+    help="Adapts protostar to existing project",
+    action="store_true",
+)
```

Comment:
I think we should avoid adding things that a user must memorize. Protostar could detect that directory is not empty when `protostar init` is run and ask user "Directory is not empty. Do you want to adapt current directory to protostar project?"

Reply:
I'd go with the solution proposed by @kasperski95, or have 2 separate commands: `new` and `init`

Reply:
>  if you create a project 99% of a time directory in which you run the script won't be empty
1) In the future, that might not be the case. This command will be rarely run, therefore it's quite likely a user will forget about this flag.

2) When a user runs `protostar init` in not empty directory, protostar should suggest `--existing` flag then.

@Arcticae @Solpatium Thoughts?

Reply:
> When a user runs protostar init in not empty directory, protostar should suggest --existing flag then.
We could print such suggestion, I like this soultion

Reply:
Two separate commands would be an overkill in my opinion, both would do basically the same

Reply:
We could specifically detect `.cairo` files (for example in 2/3 levels of depth?) to prevent some amout of false positives. We should still leave the flag though, somebody may find it usefull.